0-wrappers/FCCvote-net-neutrality/style/style.css

```diff
@@ -0,0 +1,597 @@
+html,
+body {
+  font-size: 62.5%;
+}
+body {
+  font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: 300;
+  font-size: 1.1rem;
```

Comment:
I just feel like I have to hold this so close to my face to read it:
![screen shot 2015-02-03 at 14 12 39](https://cloud.githubusercontent.com/assets/782056/6030780/c74715dc-abae-11e4-9bcb-58d46dc0d135.png)

AND YES I AM WEARING MY GLASSES